db/db.go

```diff
@@ -38,6 +38,8 @@ type KVStore interface {
 	Delete(string, []byte) error
 	// Commit commits a batch
 	Commit(KVStoreBatch) error
+	// DeleteBucket delete a Bucket
+	DeleteBucket(key []byte) error
```

Comment:
Can we avoid adding a new api. Instead, `Delete(string, []byte) error` means delete a whole bucket if the second param is not given

Reply:
sure